Source/WebCore/rendering/svg/SVGContainerLayout.cpp

```diff
@@ -72,7 +72,7 @@ void SVGContainerLayout::layoutChildren(bool containerNeedsLayout)
             if (child.isAnonymous()) {
                 ASSERT(is<RenderSVGViewportContainer>(child));
                 needsLayout = true;
-            } else if (RefPtr element = dynamicDowncast<SVGElement>(child.node())) {
+            } else if (auto element = dynamicDowncast<SVGElement>(child.protectedNode())) {
```

Comment:
Reverted this part. This may be kind of a false positive reported through [alpha.webkit.UncountedCallArgsChecker], not sure if it is worth suppressing it.